app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -112,7 +113,8 @@ def params_for_update
   end
 
   def self.params_for_create(provider)
-    capabilities = provider.capabilities.map do |name, values|
+    relevant_capabilities = provider.capabilities.select{|k| k != "data_reduction"}
+    capabilities = relevant_capabilities.map do |name, values|
```

Comment:
Prefer to use `.reject` instead of `.select` with `!=`

```suggestion
    capabilities = provider.capabilities.reject { |cap| cap == "data_reduction" }.map do |name, values|
```